examples/golang-network/docker-compose.yaml

```
@@ -0,0 +1,17 @@
+services:
+
+  proxy:
+    build:
+      context: .
+      dockerfile: ../shared/envoy/Dockerfile
+      target: envoy-go
+      args:
+        ENVOY_VARIANT: contrib-dev
+    depends_on:
+    - echo_service
+    ports:
+    - "${PORT_PROXY:-10000}:10000"
+
+  echo_service:
+    image: cjimti/go-echo
```

```suggestion
      build:
        context: .
        dockerfile: Dockerfile-echo
```

could you move this to a `Dockerfile-echo` - it just needs:

```Dockerfile
    image: cjimti/go-echo@sha256:ff9ac5cb2051d7ec0ba7b12b805550bbcb87966e9d5c288d75c18f87d9d861e9
```

you can get the latest sha with `docker buildx imagetools describe

---

done